core/controllers/pages.py

```diff
@@ -18,11 +18,13 @@
 import urlparse
 
 from core.controllers import base
-from core.controllers import editor
 from core.domain import config_domain
 import feconf
 
 
+MODERATOR_REQUEST_FORUM_URL_DEFAULT_VALUE = (
```

Comment:
I thought the same, but then later found out that it was used in the admin page.

There are actually two instances of the use of this moderator request forum link: the editor settings page (which I've just removed) and the participation playbook. I've left the second one in place because doing so would require rewriting that section of the playbook (to explain the current "featuring" workflow) and I wasn't sure what to write there.

Given this, should I leave this in, or should I remove the forum link from that page as well? If the latter, do you have any suggestions for what to replace the current content there with?